salesforcerest/salesforcerest-connector/salesforcerest-connector-1.0.0/org.wso2.carbon.connector/src/main/assembly/assemble-connector.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly>
+    <formats>
+        <format>zip</format>
+    </formats>
+
```

Comment:
Fixed in https://github.com/wso2-attic/esb-connectors/pull/559/commits/f11f75546b14c6830220ea4ae9684da0dab1ab3f